docs/source/guides/contributing.rst

```diff
@@ -1,16 +1,12 @@
 Contributing to ModdingDocs
-===========================
-
-All contributions to ModdingDocs are welcome. To add a change simply make a pull request to the `ModdingDocs repo <https://github.com/R2Northstar/ModdingDocs/>`_.
+=====
```

Comment:
What's with the changes in `contributing.rst`? They seems to be mostly formatting related. Should be separate PR IMO.

Reply:
Since the rst cheatsheet was originally following the contributing section. All the previous header organizations was incorrect and broken. So I fixed it. Then I was told to make it into another file.

Reply:
Uh can we maybe turn that into a separate PR then and keep this file here as is in this PR?

Separating things out helps with reviewing and makes reverting changes easier if need be ^^